src/hotspot/share/opto/loopnode.hpp

```diff
@@ -1117,6 +1112,7 @@ class PhaseIdealLoop : public PhaseTransform {
   // or only verify that the graph is valid if verify_me is null.
   PhaseIdealLoop(PhaseIterGVN& igvn, const PhaseIdealLoop* verify_me = nullptr) :
     PhaseTransform(Ideal_Loop),
+    _loop_or_ctrl(igvn.C->comp_arena()),
```

Comment:
Note: `_loop_or_ctrl` used to be on thread `ResourceArea`. This is bad, if we ever modify it during loopopts, inside the scope of a `ResourceMark`. I just wittnessed such a case because of the new `ResourceMark` in `SuperWord::dependence_graph.`